### PR TITLE
Josm 10526

### PR DESCRIPTION
**Pull request by Andygol**
#### Editing an existing cask
- [x] Commit message includes cask’s name (and new version, if applicable).
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.

### CODE REVIEW

**Comment by vitorgalvao** [comment]
There was a [pull request template](https://github.com/blog/2111-issue-and-pull-request-templates) when you submitted. It must be filled.

If you were not presented a template, you may have submitted via a CLI tool (likely [`hub`](https://hub.github.com/)). It that was the case, you may wish to see if the tool has a way to let you use the template (`hub` [is working on it](https://github.com/github/hub/issues/1118)).

In either case, please [refer to the template](https://raw.githubusercontent.com/caskroom/homebrew-cask/master/.github/PULL_REQUEST_TEMPLATE.md) and update your comment.

By filling the template, you’ll see what’s breaking.

File: Casks/josm.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 cask 'josm' do
-  version '10327'
+  version '10526'
   sha256 '33d1705634c554327bde3d3689723cc950ad0c42e8c47914bae92e3efce460e2'
```
Comment (vitorgalvao) [inline Casks/josm.rb]: `sha256` also needs to be updated. You can find what it is by running `shasum -a 256` on the downloaded file.

**Comment by Andygol** [comment]
@vitorgalvao 
Thank you for you help in my attempt to change JOSM cask.

But something happened in JOSM building system. There is absent OSX package - https://josm.openstreetmap.de/ticket/13134 So I'm waiting for repairing

**Comment by Andygol** [comment]
https://travis-ci.org/caskroom/homebrew-cask/jobs/144274045#L833 
>audit for josm: passed

https://travis-ci.org/caskroom/homebrew-cask/jobs/144274045#L836
>1 file inspected, no offenses detected

**Comment by Andygol** [comment]
@fanquake @vitorgalvao 
could you merge?

**Pull request merged**